Training web/Task01-HTML & CSS/style.css

```diff
@@ -0,0 +1,63 @@
+
+body ul {
+	list-style-type: none;
+}
+body li {
+	float: left;
+	padding-right: 40px;
+	font-family: 'Montserrat', sans-serif;
+	max-width: 300px;
+	margin: auto;
+	font-size: 10px;
+}
+
+
+li img {
+	float: left;
+	padding-right: 20px;
+	max-block-size: 150px;
+	object-fit: cover;
+}
+
+li p {  
+	text-align: justify;
+	text-justify: inter-word;
+}
+
+hr {
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #ccc;
+    margin: 1em 0;
+    padding: 0;
+}
+
+h2 {
+	display: inline-block;
+	padding-right: 20px;
+}
+
+button {
+	font-family: 'Montserrat', sans-serif;
+	padding: 10px 25px 10px 25px;
```

Comment:
You can do this and it's equivalent: `padding: 10px 25px;`

Reply:
Check https://www.w3schools.com/cssref/pr_padding.asp 

start reading at `If the padding property has four values:`